task01.rb

```diff
@@ -0,0 +1,10 @@
+#1. Write a script that returns true if a string contains comparison, e.g. "<23", ">45".
+
+def task01(str)
```

Comment:
@MaraEfimova We are not looking for the most performant solution, but just wanted to make it clear: you could use a couple operations instead of complex regex only if it is one (or maximum tow) very simple additional operation that will simplify the regex A LOT. For example - if you have a string that is divided in 3 parts by `-` (e.g. `asd-qwe-rty`), but there should not be a `.` (e.g. `as.d-qwe-rty`) inside each part (plus some other simple conditions), then checking for a dot inside each of the parts via regex is going to be complex and slow, instead you could first check for a dot inside the whole string (e.g. `str.include?('.')`) and then do the regex only for the other simple conditions.